library/ssl_cli.c

```diff
@@ -1717,8 +1717,12 @@ static int ssl_parse_supported_point_formats_ext( mbedtls_ssl_context *ssl,
             p[0] == MBEDTLS_ECP_PF_COMPRESSED )
         {
 #if defined(MBEDTLS_ECDH_C) || defined(MBEDTLS_ECDSA_C)
+#if defined(MBEDTLS_SSL_PROTO_TLS1_3_EXPERIMENTAL)
+            ssl->handshake->ecdh_ctx[0].point_format = p[0];
```

Comment:
We should improve readability here and have a getter which returns the ECDH context. Can you modify the commit accordingly?